src/elements/common/nav-button/NavButton.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @flow
+ * @file Nav Button component intended to mimic React Router's NavLink component for non-anchor elements
+ * @author Box
+ */
+
+import * as React from 'react';
+import classNames from 'classnames';
+import { Route } from 'react-router-dom';
+import PlainButton from '../../../components/plain-button';
+
+type Props = {
+    activeClassName?: string,
+    children: React.Node,
+    className?: string,
+    component?: any,
+    exact?: boolean,
+    onClick?: (event: SyntheticEvent<>) => void,
+    replace?: boolean,
+    strict?: boolean,
+    to: string | Location,
+};
+
+const NavButton = ({
+    activeClassName = 'bdl-is-selected',
```

Comment:
I'm not a big fan of sharing class names as constants across components, since it breaks encapsulation. This class in particular also isn't really meant to share styles with anything else; it's a modifier for this specific component. I could rename it to something like `bcs-Versions-Item-is-selected` or `bcs-VersionsItem--selected`, instead?

Reply:
that makes sense, both those alternatives seem good to me!